avm/res/key-vault/vault/secret/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.25.53.49325",
-      "templateHash": "13431676635055953332"
+      "version": "0.25.3.34343",
```

Comment:
Please note, you're not using the latest Bicep version

Reply:
fixed